### ISSUE TITLE
NG Input: Expand on hover in shortcut preferences

### ISSUE THREAD

**Issue by elstoc** [dterrahe/darktable#7]
I find the auto-expansion of the treeview on hover in preferences/shortcuts very annoying. Can we keep it to "click-to-expand" please.

Ref: darktable-org#8078

**Comment by dterrahe** [comment]
I agree with you. By default click-to-expand only responds to the little triangle all the way to the left. Which is also not great. 

Allowing click anywhere to expand/collapse I believe needs a bit of code; I'll implement that.
